test/controllers/Commissions/CommissionsController.test.ts

```diff
@@ -122,4 +124,37 @@ describe("CommissionsController", () => {
       expect(expectedCommissions).toEqual(expect.arrayContaining(actualCommissions));
     });
   });
+
+  describe("PUT /commissions", () => {
+    const expectAttributeNotDefinedError = async (attributeName: string) => {
```

Comment:
Lo mismo aca, se podria escribir esta logica dentro del mismo `it` ya que solo se usa una vez esta funcion